src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+
+import { Filter } from '../../types/Filter';
+import { GlobalContext } from '../../providers';
+
+export const TodoFooter: React.FC = () => {
+  const {
+    todos,
+    filter,
+    setFilter,
+    handleDelete,
+  } = useContext(GlobalContext);
+
+  const counter: number = todos.filter(todo => !todo.completed).length;
```

Comment:
Try to write appropriate name of the variable 
```suggestion
  const activeTodoCount = todos.filter(todo => !todo.completed).length;
```